feat: add staking apr section

neitdung opened the pull request.

neitdung committed:
add staking apr section

neitdung committed:
add catch error

neitdung committed:
extend suffix length

jiujiteiro commented on src/components/home/data_staking/hooks.ts:
```diff
@@ -59,6 +62,23 @@ export const useDataStaking = () => {
+    fetch("https://api.realio.network/realionetwork/mint/v1/annual_provisions")
+      .then(res => res.json())
+      .then(ap => {
+        fetch("https://api.realio.network/cosmos/staking/v1beta1/pool")
```
Can we use the RPC url from the env vars? It will be different on the stage deploy. Check out the docker file i think there is a var for this exact RPC

neitdung replied:
I've added `NEXT_PUBLIC_RPC_API` variable

jiujiteiro reviewed (changes requested):
small request

neitdung committed:
add bridge message

neitdung committed:
Fix burned supply

neitdung committed:
add rpc api variable

jiujiteiro commented on src/components/home/data_staking/hooks.ts:
```diff
@@ -63,10 +63,10 @@ export const useDataStaking = () => {
       .then(res => res.json())
       .then(ap => {
-        fetch("https://api.realio.network/cosmos/staking/v1beta1/pool")
+        fetch(`${process.env.NEXT_PUBLIC_RPC_API}/cosmos/staking/v1beta1/pool`)
```
@neitdung please make sure its added to the docker file and also the workflows (see .github directory)

neitdung committed:
add action variable

neitdung committed:
fix evm tx seach case

neitdung commented:
Hi @jiujiteiro, in this PR, I've implemented more following changes:

- Added the bridge message
- Fixed RIO supply
- Resolved search functionality (current Hasura doesn't support Cosmos transaction search with case variations)
